dlpy/speech.py

```diff
@@ -0,0 +1,258 @@
+from dlpy.speech_utils import *
+from dlpy.audio import AudioTable
+from dlpy.model import Model
+from dlpy.utils import get_server_path_sep, get_cas_host_type, caslibify
+import os
+import platform
+
+
+class Speech:
+    acoustic_model = None
+    language_model_name = "languageModel"
+    language_model_caslib = None
+
+    data_path = None
+    local_path = None
+
+    data_caslib = None
+    data_caslib_path = None
+    data_path_after_caslib = None
+
+    audio_table = None
+
+    def __init__(self, conn,
+                 data_path, local_path=None,
+                 acoustic_model_path=None, language_model_path=None):
+        """
```

Comment:
It's better to put the docstring on the class rather than the constructor.  You should probably have a better one-line summary of the class rather than just "Initialize".